app/views/pvb1_paths/status.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :header, 'Visit expired' %>
+
+<div class='Grid'>
+  <div class='Grid-2-3'>
+    <article>
+      <p>This visit has expired and its information has been removed.<p>
```

Comment:
@tatyree 
This a pvb1 link, so has no locale param in the url. It would default to english if we used localisation. Given that the page is not going to be translated I decided to put the copy on the view to make it explicit that this copy is only in english.

I originally had changed `en.yml` and `cy.yml` so happy to move it to `en.yml` if it makes more sense.